crates/configuration/src/version3.rs

```diff
@@ -118,12 +91,12 @@ pub struct ConfigureOptions {
 impl Default for ConfigureOptions {
     fn default() -> ConfigureOptions {
         ConfigureOptions {
-            excluded_schemas: version1::default_excluded_schemas(),
+            excluded_schemas: default_excluded_schemas(),
             unqualified_schemas: vec![],
```

Comment:
Another desired change for version 3 is to remove `unqualified_schemas`. This can happen in this PR or a follow-up.

Reply:
And in the same vein it'd be good to rename `excluded_schemas` to `excluded_schemas_for_tables` or similar.

Reply:
Yup, I already made the change but wanted to keep this diff smaller.